### DISCUSSION TITLE
How do I change only the static folder path?

### DISCUSSION THREAD

**Discussion by harebuus** [vercel/next.js#53557] [Help] [answered]
### Summary

I want my project to support multiple routes. So I need to leave the base route blank. My project is running on openshift. Another project is running on the main route. I can redirect to my project with different route routings on Openshift. My project cannot access the /_next/static/chunks folder because another project is running on the main route.
I try the /_next/static/chunks folder to work as if it were under a route I created the folder for. When I change the basePath in the nextconfig file, I cannot support the route I want, but the path of the /_next/static folder changes correctly. On the other hand, when I change the assetPrefix, the /_next/static folder is not moved to where I want it.

How do I change the path of the /_next/static folder without changing the basePath?

**Comment by Nadeera3784** [comment]
Hi @harebuus,
Did you try with changing the assetPrefix in config?

for example : 

```
module.exports = {
  // Set the assetPrefix to the value from the .env file
  assetPrefix: process.env.ASSET_PREFIX || '',
};

you can keep this ASSET_PREFIX in the env 

ASSET_PREFIX=/your-desired-path/_next/static
```

**Reply by harebuus** [reply]
I tried AssetPrefix but this value only changes the route where the files are called from the frontend. The static folder remains in the base directory. When I change this the app still can't find the static folder.

**Comment by harebuus** [comment, marked as answer]
https://github.com/vercel/next.js/issues/5602#issuecomment-673382891

This solutions works for me.

**Comment by rdizio** [comment]
This is not a proper solution. We should have the option to change the static path. Changing the folder "_next/static"  should be an option if you want a more flexible environment.

**Comment by RealFascinated** [comment]
Is there no way to change "_next/static"?

**Reply by rdizio** [reply]
currently, there is not a way to do it
